src/components/bottom-sheet/BottomSheetContext.tsx

```diff
@@ -69,18 +68,14 @@ export const BottomSheetProvider: React.FC = ({children}) => {
   const close = () => {
     setContentFunction(() => () => null);
     setIsOpen(false);
-    if (closeRef) {
-      setTimeout(() => giveFocus(closeRef), 200);
-    }
+    onCloseFocusRef && giveFocus(onCloseFocusRef);
```

Comment:
`onCloseFocusRef` should always be set, it is the `current` value which is optional.

Reply:
true!